tests/query.rs

```diff
@@ -3,30 +3,16 @@ use nickel_lang::term::{MetaValue, SharedTerm, Term};
 
 #[test]
 pub fn test_query_metadata_basic() {
-    let mut program =
-        Program::new_from_source("(1+1) | doc \"Test basic\"".as_bytes(), "regr_tests").unwrap();
-    let result = program.query(None).unwrap();
-
-    if let Term::MetaValue(meta) = result {
-        assert_eq!(meta.doc, Some(String::from("Test basic")));
-        assert_eq!(meta.value.unwrap().term, SharedTerm::new(Term::Num(2.0)));
-    } else {
-        panic!();
-    }
-}
-
-#[test]
-pub fn test_query_metadata_from_func() {
```

Comment:
The diff looks strange, but I actually kept the metdata_basic test (just put everything inside a record) and removed the from_func one. The latter tests the propagation of metadata through function application, but RFC005 decides that this won't happen anymore, hence the test isn't relevant